code/game/atoms_health.dm

```diff
@@ -0,0 +1,310 @@
+/atom
+	/// Current health for health processing. Use `get_current_health()`, `damage_health()`, or `restore_health()` for general health references.
+	var/health_current
+	/// Maximum health for simple health processing. Use `get_max_health()` or `set_max_health()` to reference/modify.
+	var/health_max
+	/// Whether or not the atom is considered 'dead'. Use `is_alive()` for general checks.
```

Comment:
Isn't this a bit redundant in view of `stat`?

Reply:
`stat` is only a mob thing isn't it?

Reply:
Most atoms that need it have a way to track dead status though. Machines have the `BROKEN` flag, mobs have `stat == DEAD`, structures have `physically_destroyed()` etc. This seems like an additional one that isn't really going to be used for those cases.

Reply:
Removed the var and replaced it with `is_alive()` checks before and after modifying health to maintain the `handle_death_change()` proc functionality. Atom overrides for `is_alive()` will need to include checks for stat, BROKEN flags, physically_destroyed(), etc as applicable.